src/consensus/pbft/libbyz/types.h

```diff
@@ -48,7 +48,11 @@ static const uint32_t MERKLE_ROOT_SIZE = 32;
 struct ByzInfo
 {
   std::array<uint8_t, MERKLE_ROOT_SIZE> replicated_state_merkle_root;
-  int64_t ctx;
+  int64_t ctx = INT64_MIN;
+  void (*cb)(void* ctx) = nullptr;
```

Comment:
Does the callback ctx have to be a void pointer here?

Reply:
no the ctx is not a pointer that is that actual context